bootstrap_node.py

```
@@ -137,19 +172,37 @@ async def cleanup_dht():
-                                            # 토픽은 유지하고 타임스탬프만 업데이트
+                                        try:
+                                            peers = json.loads(topic_data)
+                                            # 여기서 더 정교한 검증 로직 추가 가능
+                                            if peers and isinstance(peers, list):
+                                                # 토픽은 유지하고 타임스탬프만 업데이트
+                                                timestamp_data[key] = current_time
+                                                continue
+                                        except json.JSONDecodeError:
+                                            # JSON 파싱 실패 시 제거 대상으로
+                                            pass
+                                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)